lib/active_merchant/billing/gateways/xpay.rb

```diff
@@ -112,8 +107,8 @@ def add_capture_type(post, options, action)
 
       def add_refund_capture_params(amount, post, options)
         post[:amount] = amount
-        post[:currency] = options[:order][:currency]
-        post[:description] = options[:order][:description]
+        post[:currency] = options[:currency]
```

Comment:
❓ Question:

Isn't a problem the possibility that `description` or `currency` is not present?

Reply:
Right now I cannot confirm it because although the documentation says that they are optional fields, the Nexi Xpay SandBox is not working and there is no way to verify it.

Reference: https://developer.nexi.it/en/api/post-operations-operationId-refunds

Reply:
After check with the SandBox working well we have:`currency` is mandatory, `description` is not mandatory, I will remove `description` field